cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/ReplicationThrottleHelper.java

```diff
@@ -37,10 +37,10 @@
 class ReplicationThrottleHelper {
```

Comment:
There's a ZK-related comment in this file's test. 
https://github.com/linkedin/cruise-control/blob/27c0bf4fcae9583a4bd1efad140ad47d44ba21a6/cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/executor/ReplicationThrottleHelperTest.java#L118

Should we modify/remove it?

Reply:
Yes, this comment seems a leftover for me, so I changed it, thanks!
It seems to me that in the following commit (~3years ago):
Update ReplicationThrottleHelper to use AdminClient (#1781)
The test was changed to use admin client instead of zk client, but the comment remained zk related.